src/lib/prov/commoncrypto/commoncrypto_hash.cpp

```diff
@@ -71,8 +71,14 @@ class CommonCrypto_HashFunction final : public HashFunction
    private:
       void add_data(const uint8_t input[], size_t length) override
          {
-         if(m_info.update(&m_ctx, input, length) != 1)
-            throw CommonCrypto_Error("CC_" + m_info.name + "_Update");
+         /* update len parameter is 32 bit unsigned integer, feed input in parts */
+         while (length > 0)
+            {
+            CC_LONG update_len = (length > 0xFFFFFFFFUL) ? 0xFFFFFFFFUL : (CC_LONG) length;
+            m_info.update(&m_ctx, input, update_len);
```

Comment:
Missing check of return value?

Reply:
The check was removed on purpose, see the initial comment above, in particular the part "There are no errors of any kind which can be, or are, reported here,
so you can safely ignore the return values of all of these functions
if you are implementing new code."

Reply:
OK thanks